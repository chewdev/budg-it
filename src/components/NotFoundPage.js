import React from 'react';
import { Link } from 'react-router-dom';

const NotFoundPage = ({ location }) => (
	<div>
	  <h3>
		<p>404: No match for <code>{location.pathname}</code></p>
		<Link to="/">Go Home</Link>
	  </h3>
	</div>
);

export default NotFoundPage;