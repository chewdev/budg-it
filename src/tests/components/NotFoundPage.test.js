import React from 'react';
import { shallow } from 'enzyme';
import NotFoundPage from '../../components/NotFoundPage';

test('should properly render the NotFoundPage component', () => {
    const wrapper = shallow(<NotFoundPage location={ {pathname: "/wrongpage"} } />);
    expect(wrapper).toMatchSnapshot();
});