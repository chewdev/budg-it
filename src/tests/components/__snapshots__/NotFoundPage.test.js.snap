// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should properly render the NotFoundPage component 1`] = `
<div>
  <h3>
    <p>
      404: No match for 
      <code>
        /wrongpage
      </code>
    </p>
    <Link
      replace={false}
      to="/"
    >
      Go Home
    </Link>
  </h3>
</div>
`;
